ci/tasks/deploy_gcp.yaml

```diff
@@ -0,0 +1,28 @@
+platform: linux
+image_resource:
+  type: docker-image
+  source:
+    repository: google/cloud-sdk
+    tag: alpine
+
+params:
+  SERVICE_ACCOUNT_JSON: ((gcp.service_account_json))
+  PROJECT_ID:
+
+inputs:
+- name: ons-design-system-release-artifact
+
+run:
+  path: bash
+  args:
+    - -exc
+    - |
+      export GOOGLE_APPLICATION_CREDENTIALS=~/gcloud-service-key.json
+      cat >$GOOGLE_APPLICATION_CREDENTIALS <<EOL
+      $SERVICE_ACCOUNT_JSON
+      EOL
+      
+      gcloud auth activate-service-account --key-file $GOOGLE_APPLICATION_CREDENTIALS
+      gcloud config set project "${PROJECT_ID}"
```

Comment:
Is this even needed?. Buckets are globally scoped. I think we should pass in the BUCKET_NAME instead of the project id.

Reply:
I hadn't really thought about it, I assumed all gcloud commands required the project to be set. Although the bucket name is global, it still exists within a project. I'll try it without and see what happens.